File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        total_product = 1
        for num in nums:
            total_product *= num
        return_lst = []
        for num in nums:
            return_lst.append(total_product / num)
        return return_lst

File: Data Structures & Algorithms/single-number/submission-1.cpp
class Solution {
public:
    int singleNumber(std::vector<int> &nums) {
        int once = nums[0];

        for (std::size_t i = 1; i < nums.size(); ++i) {
            once ^= nums[i];
        }

        return once;
    }
};
